runtime/runtime/request.go

```diff
@@ -30,6 +33,7 @@ var (
 )
 
 var json = jsoniter.Config{
+	IndentionStep:          2,
```

Comment:
What does this change do?

Reply:
Changes the formatting of the output to be pretty formated, rather than minimal (i.e. so the output looks the same as our hand written JSON responses)